packages/pwa-devdocs/README.md

```diff
@@ -3,66 +3,71 @@
 This is the source code and documentation repository for the PWA Studios [documentation site].
 
 The PWA DevDocs site is build using [Jekyll] and hosted using [GitHub Pages].
- 
+
 ## Getting Started
- 
+
 Use the following instructions to build a local instance of the documentation.
- 
+
 ### Prerequisites
- 
-* [Git]
-* [NodeJS]
-* [Ruby]
-* [Bundler]
-* [Yarn]
- 
+
+-   [Git]
+-   [NodeJS]
+-   [Ruby]
+-   [Bundler]
+-   [Yarn]
```

Comment:
Did you mean `npm`? I don't see `yarn` usage anywhere here

Reply:
A bit curious what Ruby is used for.

Reply:
It's used in `pwa-devdocs` to run Jekyll.

Reply:
@DrewML 
I can change it to `npm` since the rest of the repo uses it